packages/components/src/internal/actions.ts

```diff
@@ -762,7 +762,9 @@ export function getExportParams(
 
     if (options) {
         if (options.columns) {
-            params['query.columns'] = options.columns;
+            if (advancedOptions && advancedOptions['includeColumn'])
+                params['query.columns'] = options.columns + ',' + advancedOptions['includeColumn'].join(',');
+            else params['query.columns'] = options.columns;
```

Comment:
Code styling here is a little non-standard, I think.

Reply:
this appears to be done by lint